### ISSUE TITLE
Quick (sub-)filter on list of transactions

### ISSUE THREAD

**Issue by sarjanwebdev** [moneymanagerex/moneymanagerex#5657]
Once the Transaction Filter is applied
I sometimes need to focus on a subset of records
1. AS IS I apply a transaction filter to seek a date range, let's say it returns 100 records
2. REQUEST  I want to further zoom into records with certain keyword, which I do not want to do another transaction filter, I wish I could enter the keyword somewhere on the same screen and filter out the particular 22 records. This may not re-query the database but only the current view, hence is quicker. (Keeping HTML+JS client search vs MYSQL server search in mind)

**Comment by tactilis** [comment]
> records with certain keyword

What do you mean by 'keyword'?  Which column(s) do you want the keyword sub-filter to search in?

**Comment by sarjanwebdev** [comment]
We should be able to search Notes, Amount, Dates, Category name, Status, and Balance.

**Comment by rmelillo76** [comment]
> Is it worth considering supporting something like: https://tabulator.info/ for tables, in the same way we have ApexCharts? Will add much more flexibility
> 
> E.g. https://tabulator.info/examples/5.4#filter

Something like that would definitely add much more flexibility, but I'm not sure how feasible it is now, given the current components used within MMEX. I took the request @sarjanwebdev to refer to the transaction grids that use the wxWidgets control, not the reports area which uses HTML. But maybe he was referring to both. But even if we did want to enhance the reports, on Windows the embedded Internet Explorer web browser control wouldn't support that, I know there is a separate issue in the backlog on whether to migrate away from the IE browser control but I recall that being a significant effort since Microsoft doesn't support in memory content. Just a few things to consider.

**Comment by whalley** [comment]
Yes, would likely need to update the browser control for windows

**Comment by n-stein** [comment]
Not a fix, but #5512 might help a little?

**Comment by n-stein** [comment]
Tabulator works with the Edge backend
![mmex_rTCQJPIvXg](https://github.com/moneymanagerex/moneymanagerex/assets/16599929/72440fde-f72e-40cf-96cb-ad80e6f71773)

**Comment by sarjanwebdev** [comment]
I want similar thing on transaction list (not on report)
